occurrences/schema.py

```diff
@@ -98,9 +98,50 @@ class PalvelutarjotinEventInput(InputObjectType):
     auto_acceptance = graphene.Boolean()
 
 
-class StudyGroupNode(DjangoObjectType):
-    study_level = StudyLevelEnum()
+class StudyLevelTranslationType(DjangoObjectType):
+    language_code = LanguageEnum(required=True)
 
+    class Meta:
+        model = StudyLevelTranslation
+        exclude = ("id", "master")
+
+
+class StudyLevelTranslationsInput(InputObjectType):
+    language_code = LanguageEnum(required=True)
+
+
+class StudyLevelNode(DjangoObjectType):
+    # NOTE: Should the originalId have a better name?
+    originalId = graphene.String(source="pk")
```

Comment:
As commented below, I think we can just use `ID` to display study level ID in plain text.